client/src/pages/queries/components/create-chart.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { cloneElement, useMemo, useContext, useState } from 'react'
+import { Form, Empty, Select, Input, Button, Tooltip, message } from 'antd'
+import { useMutation, useQuery, useQueryClient } from 'react-query'
+import EditorContext from 'contexts/editor-context'
+import { ChartTypes } from 'types/chart'
+import { chartGroups, chartList, chartOrder } from '../data/chart-list'
+import { createChart, fetchChartsForQuery } from '../chart-queries'
+
+type CreateChartProps = {
+  changeActiveChartId: (id: number) => void
+}
+
+export default function CreateChart({ changeActiveChartId }: CreateChartProps) {
+  const { queryResult, isSaved, queryId } = useContext(EditorContext)
+
+  const [label, setLabel] = useState<string | undefined>()
+  const [values, setValues] = useState<string[] | undefined>()
+  const [title, setTitle] = useState<string>('Untitled Chart')
+  const [chartType, setChartType] = useState<ChartTypes>('line')
```

Comment:
Same as `chart-tab.tsx` comments use https://ant.design/components/form/#components-form-demo-customized-form-controls for handling state changes.